docs/yang/parsers.rst

```diff
@@ -183,13 +183,15 @@ the device. For example::
         parser: XMLParser
         execute:
             - method: _rpc
-              args:
+              args: []
```

Comment:
Both don't need to be specified in the YAML, but I think it might be wise to have the support in the future for `args` in the backend.